static/skywire-manager-src/src/assets/i18n/de.json

```diff
@@ -1,39 +1,78 @@
 {
   "common": {
     "save": "Speichern",
-    "edit": "Ändern",
     "cancel": "Abbrechen",
-    "node-key": "Visor Schlüssel",
-    "app-key": "Anwendungs-Schlüssel",
-    "discovery": "Discovery",
     "downloaded": "Heruntergeladen",
     "uploaded": "Hochgeladen",
-    "delete": "Löschen",
-    "none": "Nichts",
     "loading-error": "Beim Laden der Daten ist ein Fehler aufgetreten. Versuche es erneut...",
     "operation-error": "Beim Ausführen der Aktion ist ein Fehler aufgetreten.",
     "no-connection-error": "Es ist keine Internetverbindung oder Verbindung zum Hypervisor vorhanden.",
     "error": "Fehler:",
     "refreshed": "Daten aktualisiert.",
     "options": "Optionen",
     "logout": "Abmelden",
-    "logout-error": "Fehler beim Abmelden."
+    "logout-error": "Fehler beim Abmelden.",
+    "logout-confirmation": "Wirklich abmelden?",
+    "time-in-ms": "{{ time }}ms",
+    "ok": "Ok",
+    "unknown": "Unbekannt",
+    "close": "Schließen"
   },
 
-  "tables": {
-    "title": "Ordnen nach",
-    "sorting-title": "Geordnet nach:",
-    "ascending-order": "(aufsteigend)",
-    "descending-order": "(absteigend)"
+  "labeled-element": {
+    "edit-label": "Bezeichnung ändern",
+    "remove-label": "Bezeichnung löschen",
+    "copy": "Kopieren",
+    "remove-label-confirmation": "Bezeichnung wirklich löschen?",
+    "unnamed-element": "unbenannt",
```

Comment:
Should we capitalize that? It looks slightly odd in the UI if it is not imo.

Reply:
done